chart/values.yaml

```diff
@@ -323,7 +328,7 @@ csi:
       keep_alive_tmo: ""
     # -- The kubeletDir directory for the csi-node plugin
     kubeletDir: /var/lib/kubelet
-    pluginMounthPath: /csi
+    pluginMountPath: /csi
```

Comment:
Please move this typo change to separate commit.
Also @niladrih we might need to plugin upgrade code to support this?

Reply:
I'd suggest to remove it form the PR and add both typo fix and plugin code in another PR

Reply:
Sure

Reply:
Yes we don't need this change, let's remove it.

Reply:
This will need an upgrade change, yes.

Reply:
Reverted the changes

Reply:
I've raised PR with the upgrade changes required for both (1) the logging values changes, (2) the type change.
https://github.com/openebs/mayastor-extensions/pull/443.